packages/cashier/src/stores/withdraw-store.ts

```diff
@@ -1,30 +1,34 @@
-import React from 'react';
+import { ReactElement } from 'react';
 import { action, computed, observable } from 'mobx';
 import { formatMoney, getDecimalPlaces, getMinWithdrawal, isMobile, validNumber } from '@deriv/shared';
 import { localize } from '@deriv/translations';
-import { ReadMore } from '@deriv/components';
+import { CryptoConfig } from '@deriv/api-types';
+import ReadMoreWrapper from 'Components/read-more-wrapper';
 import Constants from 'Constants/constants';
 import ErrorStore from './error-store';
 import VerificationStore from './verification-store';
+import { TRootStore, TWebSocket } from 'Types';
 
 export default class WithdrawStore {
-    constructor({ WS, root_store }) {
+    constructor(public WS: TWebSocket, public root_store: TRootStore) {
         this.root_store = root_store;
         this.WS = WS;
     }
 
     @observable blockchain_address = '';
     @observable container = Constants.containers.withdraw;
     @observable error = new ErrorStore();
-    @observable is_10k_withdrawal_limit_reached = undefined;
+    @observable is_10k_withdrawal_limit_reached?: boolean = undefined;
```

Comment:
Why can't we use false as a init value here?

```suggestion
    @observable is_10k_withdrawal_limit_reached = false;
```

Reply:
in withdraw.tsx there is seprate check for false and undefined, so for now let us not remove undefined